src/reactable.jsx

```diff
@@ -519,30 +519,30 @@
                     React.Children.forEach(child.props.children, function(descendant) {
                         // TODO
                         /* if (descendant.type.ConvenienceConstructor === Td) { */
-                        if (true) {
```

Comment:
I figured that was it, but I feel weird about having an `if (true)` just as a placeholder. your call tho